testsuite/integration-arquillian/tests/base/testsuites/database-suite

```diff
@@ -0,0 +1,17 @@
+AccountRestServiceTest
```

Comment:
@mposolda @pedroigor @hmlnarik I need some help in reviewing this list and checking if it's sufficient, at least initially